### PR TITLE
XlsIO - Helper Files for MAUI, Blazor and Xamarin - HF

### PR DESCRIPTION
**Pull request by KonduruKeerthi**
Helper Files for MAUI, Blazor and Xamarin

### CODE REVIEW

File: File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-maui-c-sharp.md [hunk 1]
```diff
@@ -59,6 +59,14 @@ workbook.SaveAs(outputStream);
 outputStream.Position = 0;
 
 //Saves the memory stream as a file.
-DependencyService.Get<ISave>().SaveAndView("Output.xlsx", "application/excel", outputStream);
+SaveService saveService = new SaveService();
+saveService.SaveAndView("Output.xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", ms);
 {% endhighlight %}
 {% endtabs %} 
+
+The code for SaveService helper class is present in below pages.
```
Comment (Mohan2401) [inline File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-maui-c-sharp.md:67]: can we import the code from getting started instead links similar to Blazor.

File: File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-xamarin-c-sharp.md [hunk 2]
```diff
@@ -62,3 +62,93 @@ outputStream.Position = 0;
 Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Output.xlsx", "application/msexcel", outputStream);
 {% endhighlight %}
 {% endtabs %} 
+
+Download the helper files from this [link](https://www.syncfusion.com/downloads/support/directtrac/general/ze/HELPER~1-1423062113.zip) and add them into the mentioned project. These helper files allow you to save the stream as a physical file and open the file for viewing.
```
Comment (Mohan2401) [inline File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-xamarin-c-sharp.md:66]: Add save code here too.

**Review by Mohan2401 — CHANGES_REQUESTED** [review]

File: File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-maui-c-sharp.md (hunk 1 above)
Comment (KonduruKeerthi) [reply File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-maui-c-sharp.md]: @Mohan2401 - Made changes as suggested

File: File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-xamarin-c-sharp.md (hunk 2 above)
Comment (KonduruKeerthi) [reply File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-xamarin-c-sharp.md]: @Mohan2401  - Made changes as suggested

**Review by Mohan2401 — APPROVED** [review]

**Pull request merged**